### PR TITLE
Copyedits in "Operating System Regional Data Display"

### PR DESCRIPTION
**Pull request by JasonAndrewWriter**
Fixed ICU link and table.

**Rendered article section for review:**
https://review.docs.microsoft.com/en-us/microsoft-edge/web-platform/os-regional-settings?branch=pr-en-us-2134

### CODE REVIEW

**Comment by JasonAndrewWriter** [comment]
#reviewer

File: microsoft-edge/web-platform/os-regional-settings.md [hunk 1]
```diff
@@ -29,15 +29,15 @@ Websites can use the built-in Internationalization API with a regional code to p
...
 
 To set the user's language preferences, Microsoft Edge gets the default values from the OS when it is first installed. The user can then change the language preferences in `edge://settings/languages` and sort them in priority order.
 
 | Language or region data | User impact |
 |--------|-------------------------|
 | **Preferred language** in the **setting** page | The language selected for rendering Microsoft Edge. This language/region is also used for formatting values such as date, time, and numbers. |
-|  `accept-language` HTTP header | A list of regional codes sent to the server to allow a Website to display the right language. Microsoft Edge sends the regional codes in the order specified in **edge://settings/languages**. |
-| `navigator.language` API (and similar) | Allow a Website to display the right language and regional formatting for user input and other content produced locally. |
+|  `accept-language` HTTP header | A list of regional codes (as defined in preferred languages) sent to the server to allow a Website to display the right language. Microsoft Edge sends the regional codes in the order specified in **edge://settings/languages**. |
```
Comment (mikehoffms) [inline microsoft-edge/web-platform/os-regional-settings.md:39]: the actual UI text, which should somehow be shown in bold here:
The "**Settings**" page > "**Languages**" page (selected on left) opens the Languages page within Settings (url is edge://settings/languages ) > the "**Preferred languages**" dropdown list, or dropdown menu.
```suggestion
|  `accept-language` HTTP header | A list of regional codes (as defined in **Preferred languages**) sent to the server to allow a Website to display the right language. Microsoft Edge sends the regional codes in the order specified in **edge://settings/languages**. |
```
image

File: microsoft-edge/web-platform/os-regional-settings.md [hunk 2]
```diff
@@ -29,15 +29,15 @@ Websites can use the built-in Internationalization API with a regional code to p
...
 
-A regional code is a two-letter language code followed by a hyphen and a two-letter region code.  Microsoft Edge uses ICU to understand regional codes. ICU is an implementation of the Unicode standard's Internationalization API.  See [ICU](https://icu.unicode.org/home).
+A regional code is a two-letter language code followed by a hyphen and a two-letter region code.  Microsoft Edge uses ICU to understand regional codes. ICU is an implementation of the Unicode standard's Internationalization API.  For more information, see [ICU-TC Home Page](https://icu.unicode.org/home).
 
 To set the user's language preferences, Microsoft Edge gets the default values from the OS when it is first installed. The user can then change the language preferences in `edge://settings/languages` and sort them in priority order.
 
 | Language or region data | User impact |
 |--------|-------------------------|
 | **Preferred language** in the **setting** page | The language selected for rendering Microsoft Edge. This language/region is also used for formatting values such as date, time, and numbers. |
```
Comment (mikehoffms) [inline microsoft-edge/web-platform/os-regional-settings.md:38]:
```suggestion
| **Preferred languages** in the **Settings** > **Languages** page | The language selected for rendering Microsoft Edge. This language/region is also used for formatting values such as date, time, and numbers. |
```
Check UI labels.  On my screen, has an 's' and the list is "English (United States)" and then below it, "English".  AFAIK, these are my defaults.

File: microsoft-edge/web-platform/os-regional-settings.md [hunk 3]
```diff
@@ -29,15 +29,15 @@ Websites can use the built-in Internationalization API with a regional code to p
...
-|  `accept-language` HTTP header | A list of regional codes sent to the server to allow a Website to display the right language. Microsoft Edge sends the regional codes in the order specified in **edge://settings/languages**. |
-| `navigator.language` API (and similar) | Allow a Website to display the right language and regional formatting for user input and other content produced locally. |
+|  `accept-language` HTTP header | A list of regional codes (as defined in preferred languages) sent to the server to allow a Website to display the right language. Microsoft Edge sends the regional codes in the order specified in **edge://settings/languages**. |
+| `navigator.language` API (and similar) | Code inserted into a website that allows the browsers to display the correct regional formatting (as defined in preferred languages) for user input and other content produced locally. |
```
Comment (mikehoffms) [inline microsoft-edge/web-platform/os-regional-settings.md:40]:
```suggestion
| `navigator.language` API (and similar) | Code inserted into a website that allows the browsers to display the correct regional formatting (as defined in  **Settings** > **Languages** > **Preferred languages**) for user input and other content produced locally. |
```

**Review by mikehoffms — CHANGES_REQUESTED** [review]
I think the article intends to specify these UI controls per my comments

File: microsoft-edge/web-platform/os-regional-settings.md (hunk 1 above)
Comment (mikehoffms) [reply microsoft-edge/web-platform/os-regional-settings.md:39]: I really think another screenshot might be needed, like pasted into my previous comment, showing **Languages** nav on left, and the **Preferred languages** section.

Comment (JasonAndrewWriter) [reply microsoft-edge/web-platform/os-regional-settings.md]: I just added a new image

File: microsoft-edge/web-platform/os-regional-settings.md [hunk 4]
```diff
@@ -29,15 +29,15 @@ Websites can use the built-in Internationalization API with a regional code to p
 
 In general, the website's internationalization user experiences are only as good as the regional code that they're able to determine for the current user. 
 
-A regional code is a two-letter language code followed by a hyphen and a two-letter region code.  Microsoft Edge uses ICU to understand regional codes. ICU is an implementation of the Unicode standard's Internationalization API.  See [ICU](https://icu.unicode.org/home).
+A regional code is a two-letter language code followed by a hyphen and a two-letter region code.  Microsoft Edge uses ICU to understand regional codes. ICU is an implementation of the Unicode standard's Internationalization API.  For more information, see [ICU-TC Home Page](https://icu.unicode.org/home).
 
 To set the user's language preferences, Microsoft Edge gets the default values from the OS when it is first installed. The user can then change the language preferences in `edge://settings/languages` and sort them in priority order.
```
Comment (mikehoffms) [inline microsoft-edge/web-platform/os-regional-settings.md:34]: pls move the new "Preferred languages" screenshot to near here (it's mentioned in this sentence & table, maybe elsewhere too), and restore the "OS regional" screenshot.  Note that in the UI, "Preferred lang" is at top of UI and "OS region" is at bottom.  I looked at them both at once onscreen in the UI but they are spread too far apart to fit into a single capture at 100% zoom level.  They are not adjacent in the UI else single capture would be perfect.

Comment (JasonAndrewWriter) [reply microsoft-edge/web-platform/os-regional-settings.md:34]: done

Comment (JasonAndrewWriter) [reply microsoft-edge/web-platform/os-regional-settings.md:34]: We're moving into styleist changes, but yes I did this one.

**Review by mikehoffms — APPROVED** [review]

**Pull request merged**